### ISSUE TITLE
Crash server with 1.6.1 and 1.6.0

### ISSUE THREAD

**Issue by MisterJack49** [MineMaarten/PneumaticCraft#411]
When updating PneumaticCraft to latest version on my server, the following error occures : 
```java
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 20/02/15 15:04
Description: Exception in server tick loop

cpw.mods.fml.common.MissingModsException
	at cpw.mods.fml.common.Loader.sortModList(Loader.java:244)
	at cpw.mods.fml.common.Loader.loadMods(Loader.java:472)
	at cpw.mods.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:87)
	at cpw.mods.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:314)
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Linux (amd64) version 3.12-0.bpo.1-amd64
	Java Version: 1.7.0, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 280895008 bytes (267 MB) / 518979584 bytes (494 MB) up to 18407424000 bytes (17554 MB)
	JVM Flags: 21 total; -Xms512M -XX:PermSize=384m -XX:MaxPermSize=768m -XX:-UseConcMarkSweepGC -XX:+UseParNewGC -XX:+UseNUMA -XX:+CMSParallelRemarkEnabled -XX:+UseAdaptiveGCBoundary -XX:-UseGCOverheadLimit -XX:+UseBiasedLocking -XX:SurvivorRatio=8 -XX:TargetSurvivorRatio=90 -XX:MaxTenuringThreshold=15 -XX:UseSSE=3 -XX:+UseLargePages -XX:+UseFastAccessorMethods -XX:+UseStringCache -XX:+UseCompressedOops -XX:+OptimizeStringConcat -XX:+AggressiveOpts -XX:ParallelGCThreads=1
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.85.1230 Minecraft Forge 10.13.2.1230 43 mods loaded, 43 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded
	FML{7.10.85.1230} [Forge Mod Loader] (forge-1.7.10-10.13.2.1230-universal.jar) Unloaded
	Forge{10.13.2.1230} [Minecraft Forge] (forge-1.7.10-10.13.2.1230-universal.jar) Unloaded
	appliedenergistics2-core{rv2-beta-1} [AppliedEnergistics2 Core] (minecraft.jar) Unloaded
	CodeChickenCore{1.0.4.29} [CodeChicken Core] (minecraft.jar) Unloaded
	<CoFH ASM>{000} [CoFH ASM Data Initialization] (minecraft.jar) Unloaded
	MobiusCore{1.2.3} [MobiusCore] (minecraft.jar) Unloaded
	NotEnoughItems{1.0.3.73} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.3.73-universal.jar) Unloaded
	OpenModsCore{0.6} [OpenModsCore] (minecraft.jar) Unloaded
	appliedenergistics2{rv2-beta-1} [Applied Energistics 2] (appliedenergistics2-rv2-beta-1.jar) Unloaded
	BigReactors{0.4.0A} [Big Reactors] (BigReactors-0.4.0A.jar) Unloaded
	BuildCraft|Silicon{6.2.6} [BC Silicon] (buildcraft-6.2.6.jar) Unloaded
	BuildCraft|Core{6.2.6} [BuildCraft] (buildcraft-6.2.6.jar) Unloaded
	BuildCraft|Transport{6.2.6} [BC Transport] (buildcraft-6.2.6.jar) Unloaded
	BuildCraft|Builders{6.2.6} [BC Builders] (buildcraft-6.2.6.jar) Unloaded
	BuildCraft|Energy{6.2.6} [BC Energy] (buildcraft-6.2.6.jar) Unloaded
	BuildCraft|Factory{6.2.6} [BC Factory] (buildcraft-6.2.6.jar) Unloaded
	ChickenChunks{1.3.4.14} [ChickenChunks] (ChickenChunks-1.7.10-1.3.4.14-universal.jar) Unloaded
	chisel{2.1.3} [Chisel] (Chisel 2-2.1.3.jar) Unloaded
	CoFHCore{1.7.10R3.0.0B9} [CoFH Core] (CoFHCore-[1.7.10]3.0.0B9-40.jar) Unloaded
	Dynmap{2.1-104} [Dynmap] (Dynmap-2.1-forge-1.7.10.jar) Unloaded
	EnderIO{1.7.10-2.2.5.311} [Ender IO] (EnderIO-1.7.10-2.2.5.311.jar) Unloaded
	IC2NuclearControl{2.0.5b-Unicorn} [Nuclear Control 2] (IC2NuclearControl-2.0.5b-Unicorn.jar) Unloaded
	IC2{2.2.648-experimental} [IndustrialCraft 2] (industrialcraft-2-2.2.648-experimental.jar) Unloaded
	IronChest{6.0.60.741} [Iron Chest] (ironchest-1.7.10-6.0.60.741-universal.jar) Unloaded
	Mantle{1.7.10-0.3.2.jenkins184} [Mantle] (Mantle-1.7.10-0.3.2.jar) Unloaded
	Mekanism{7.1.1} [Mekanism] (Mekanism-1.7.10-7.1.1.127.jar) Unloaded
	MekanismGenerators{7.1.1} [MekanismGenerators] (MekanismGenerators-1.7.10-7.1.1.127.jar) Unloaded
	MekanismTools{7.1.1} [MekanismTools] (MekanismTools-1.7.10-7.1.1.127.jar) Unloaded
	Natura{2.2.0} [Natura] (natura-1.7.10-2.2.0.1.jar) Unloaded
	OpenBlocks{1.3} [OpenBlocks] (OpenBlocks-1.7.10-1.3-snapshot-513.jar) Unloaded
	OpenMods{0.6} [OpenMods] (OpenModsLib-1.7.10-0.6-snapshot-268.jar) Unloaded
	PneumaticCraft{1.6.1-62} [PneumaticCraft] (PneumaticCraft-1.7.10-1.6.1-62-universal.jar) Unloaded
	ProjRed|Core{4.5.6.57} [ProjectRed] (ProjectRed-1.7.10-4.5.6.57-Base.jar) Unloaded
	ProjRed|Compatibility{4.5.6.57} [ProjectRed-Compatibility] (ProjectRed-1.7.10-4.5.6.57-Compat.jar) Unloaded
	ProjRed|Integration{4.5.6.57} [ProjectRed-Integration] (ProjectRed-1.7.10-4.5.6.57-Integration.jar) Unloaded
	ProjRed|Transmission{4.5.6.57} [ProjectRed-Transmission] (ProjectRed-1.7.10-4.5.6.57-Integration.jar) Unloaded
	ProjRed|Illumination{4.5.6.57} [ProjectRed-Illumination] (ProjectRed-1.7.10-4.5.6.57-Lighting.jar) Unloaded
	ProjRed|Exploration{4.5.6.57} [ProjectRed-Exploration] (ProjectRed-1.7.10-4.5.6.57-World.jar) Unloaded
	ForgeMicroblock{1.1.0.314} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.0.314-universal.jar) Unloaded
	ForgeMultipart{1.1.0.314} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.0.314-universal.jar) Unloaded
	McMultipart{1.1.0.314} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.0.314-universal.jar) Unloaded
	MrTJPCoreMod{1.0} [MrTJPCoreMod] (MrTJPCore-1.7.10-1.0.3.5-universal.jar) Unloaded
	Profiler Position: N/A (disabled)
	Is Modded: Definitely; Server brand changed to 'fml,forge'
	Type: Dedicated Server (map_server.txt)
```

**Comment by MineMaarten** [comment]
Why have you closed it? derp from your side?:P

**Comment by MisterJack49** [comment]
Yep my bad forge isn't up to date, but I cannot upgrade it myself :( (omgserv only propose v10.13.2.1230 of Forge)

**Comment by spocane** [comment]
I know that this issue is closed but i am getting this when using v10.13.2.1291 (the current recommended) of Forge and 1.6.1-62 of PnematicCraft
